lib/index.js

```diff
@@ -303,6 +303,9 @@ const parseDatasetIdentifier = async (path_) => {
         parts[1] = resolvedPath.userid
       }
       let res = await fetch(`https://api.datahub.io/source/${parts[1]}/${name}/successful`)
+      if (res.status !== 200) {
+        throw new Error(res.statusText)
```

Comment:
@Mikanebu can you please throw error in the following way:

```javascript
throw new Error({
  status: res.status,
  message: res.statusText
})
```

This way we can handle this error easily and print better messages